### ISSUE TITLE
CRUD: Get by Id

### ISSUE THREAD

**Issue by ccocker** [ericmdantas/generator-ng-fullstack#179]
To complete the CRUD operations it would be a nice enhancement to have a getById method in the todo service for example: -

**tod-do.service.ts**

```
getById(id: string): Observable<any> {

        return this._http
            .get(TodoService.ENDPOINT.replace(':id', id))
            .map((r) => r.json());
    }


```
**todo.controller.ts**

```
static getToDoById(req: express.Request, res: express.Response): void {
        let _todo = req.body;

        TodoDAO
        ['getCustomer'](_todo)
            .then(todo => res.status(201).json(todo))
            .catch(error => res.status(400).json(error));
    }

```
**todo-dao.ts**

```
todoSchema.static('getToDo', (id):Promise<any> => {
    return new Promise((resolve:Function, reject:Function) => {
        let _query = {id};

        Todo
          .find(_query)
          .exec((err, todo) => {
              err ? reject(err)
                  : resolve(todo);
          });
    });
});

```

**Comment by ericmdantas** [comment]
Targetting 1.9.

**Comment by ericmdantas** [comment]
We'll also have to update the tests to cover the new method.

**Comment by ericmdantas** [comment]
Closed, will be released in 1.9.
